dataverse-webapp/src/main/webapp/resources/vecler/theme.scss

```diff
@@ -2454,7 +2529,1175 @@ div.ui-dialog.largePopUp {
 .consent-label {
 	padding-left: 1% !important;
 }
-
+  
 .no-bottom-border {
     border-bottom: none !important;
 }
+
+
+/**
+ * HIGH CONTRAST VERSION BELOW
+ */
```

Comment:
I have some issues with submenu for selected file in datatable
![menu_selected_file_tabular](https://user-images.githubusercontent.com/8190961/78988846-5140bd80-7b32-11ea-93c0-c5dbdc8c267e.png)

Reply:
Fixed